nucleus/base/admin.py

```diff
@@ -35,5 +35,4 @@ def has_delete_permission(self, request, obj=None):
         return False
 
 
-admin.site.register(GithubLog, GithubLogAdmin)
 admin.site.register(admin.models.LogEntry, LogEntryAdmin)
```

Comment:
`django-upgrade` missed this one for me to. You can do it manually so these decorators are consistent.